src/app/business/services/workflow.service.ts

```diff
@@ -60,7 +60,7 @@ export class WorkflowService {
   public createInstance(param) {
     let detailUrl = this.url + 'instances';
     console.log("Final Param to post", param);
-    return this.http.post(detailUrl, param);
```

Comment:
Is this correlated change?

Reply:
Yes. Sending the X-Auth-Token in header.